Assets/Scripts/UI/RetryScript.cs

```diff
@@ -0,0 +1,26 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class RetryScript : MonoBehaviour {
+
+    public GameObject Retry;
+    // Use this for initialization
+    void Start()
+    {
+        Retry.SetActive(false);
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+
+    }
+
+    void OnTriggerEnter2D(Collider2D col)
+    {
+        if (col.gameObject.tag == "Player")
+        {
+            Retry.SetActive(true);
```

Comment:
Yeah, I have added in 3b since most people fail to go past the conveyer belt at that level. Bassically my idea was to add this collider in every place the player has to click retry.